src/wasm/html/css/innoextract.css

```diff
@@ -0,0 +1,13 @@
+#con {
```

Comment:
Console logs should be more tightly spaced, for example:
`#con > p { margin: 0px; }`
or by adding some kind of higher class for console text (.info and .err), like `.con-txt`